mmdet3d/ops/pointnet_modules/point_sa_module.py

```diff
@@ -108,10 +112,12 @@ def _sample_points(self, points_xyz, features, indices, target_xyz):
         Otherwise sample points using `self.points_sampler`.
 
         Args:
-            points_xyz (Tensor): (B, N, 3) xyz coordinates of the features.
-            features (Tensor): (B, C, N) features of each point.
-            indices (Tensor): (B, num_point) Index of the features.
-            target_xyz (Tensor): (B, M, 3) new_xyz coordinates of the outputs.
+            points_xyz (tensor.Tensor): (B, N, 3) xyz coordinates of the
```

Comment:
Should be `torch.Tensor` instead of `tensor.Tensor`. Also fix the other comments